integration/network/service_test.go

```diff
@@ -94,16 +73,16 @@ func TestServiceWithIngressNetwork(t *testing.T) {
 	assert.NilError(t, err)
 
 	serviceID := serviceResp.ID
-	poll.WaitOn(t, serviceRunningCount(client, serviceID, instances), pollSettings)
+	poll.WaitOn(t, serviceRunningCount(client, serviceID, instances), swarm.NetworkPoll)
 
 	_, _, err = client.ServiceInspectWithRaw(context.Background(), serviceID, types.ServiceInspectOptions{})
 	assert.NilError(t, err)
 
 	err = client.ServiceRemove(context.Background(), serviceID)
 	assert.NilError(t, err)
 
-	poll.WaitOn(t, serviceIsRemoved(client, serviceID), pollSettings)
-	poll.WaitOn(t, noServices(client), pollSettings)
+	poll.WaitOn(t, serviceIsRemoved(client, serviceID), swarm.NetworkPoll)
+	poll.WaitOn(t, noServices(client), swarm.NetworkPoll)
```

Comment:
these are waiting on services, so shouldn't it use `ServicePoll`?

Reply:
Right, I'll correct it.